### PR TITLE
Add rules-base to BOM

### PR DESCRIPTION
**Pull request by OndraZizka**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Merged build triggered.

File: bom/pom.xml [hunk 1]
```diff
@@ -162,8 +162,18 @@
...
 
+
+            <!-- Windup Rules Base -->
+            <dependency>
+                <groupId>org.jboss.windup.rules.apps</groupId>
+                <artifactId>rules-base</artifactId>
+                <classifier>forge-addon</classifier>
+                <version>${project.version}</version>
+            </dependency>
+
```
Comment (lincolnthree) [inline bom/pom.xml]: This also needs the api and impl dependencies.

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build started.

File: bom/pom.xml (hunk 1 above)
Comment (OndraZizka) [inline bom/pom.xml]: Added

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build finished. Test FAILed.

**Comment by windupgithubbot1** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://10.3.9.192//job/windup-pr-builder/407/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.config:windup-config-tests' /><a href='https://10.3.9.192/job/windup-pr-builder/407/org.jboss.windup.config$windup-config-tests/testReport'>org.jboss.windup.config:windup-config-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://10.3.9.192/job/windup-pr-builder/407/org.jboss.windup.config$windup-config-tests/testReport/org.jboss.windup.config.iteration/RuleIterationOverDefaultSingleVariableTest/testTypeSelection/'><strong>org.jboss.windup.config.iteration.RuleIterationOverDefaultSingleVariableTest.testTypeSelection</strong></a></li></ul>
Test FAILed.

**Comment by lincolnthree** [comment]
Incorporated in https://github.com/windup/windup/pull/431

**Comment by windupgithubbot1** [comment]
Merged build started.

**Comment by windupgithubbot1** [comment]
Merged build finished. Test FAILed.

**Comment by windupgithubbot1** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://10.3.9.192//job/windup-pr-builder/408/<h2>Failed Tests: <span class='status-failure'>6</span></h2><h3><a name='org.jboss.windup.reporting:windup-reporting-tests' /><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.reporting$windup-reporting-tests/testReport'>org.jboss.windup.reporting:windup-reporting-tests</a>: <span class='status-failure'>2</span></h3><ul><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.reporting$windup-reporting-tests/testReport/org.jboss.windup.reporting/InlineHintModelQueryTest/testFindingClassifiedFiles/'><strong>org.jboss.windup.reporting.InlineHintModelQueryTest.testFindingClassifiedFiles</strong></a></li><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.reporting$windup-reporting-tests/testReport/org.jboss.windup.reporting/InlineHintModelQueryTest/testFindingNonClassifiedFiles/'><strong>org.jboss.windup.reporting.InlineHintModelQueryTest.testFindingNonClassifiedFiles</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:rules-java' /><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.rules.apps$rules-java/testReport'>org.jboss.windup.rules.apps:rules-java</a>: <span class='status-failure'>3</span></h3><ul><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.rules.apps$rules-java/testReport/org.jboss.windup.rules.java/SourceModeTest/testSourceModeDefault/'><strong>org.jboss.windup.rules.java.SourceModeTest.testSourceModeDefault</strong></a></li><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.rules.apps$rules-java/testReport/org.jboss.windup.rules.java/SourceModeTest/testSourceModePositive/'><strong>org.jboss.windup.rules.java.SourceModeTest.testSourceModePositive</strong></a></li><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.rules.apps$rules-java/testReport/org.jboss.windup.rules.java/SourceModeTest/testSourceModeNegative/'><strong>org.jboss.windup.rules.java.SourceModeTest.testSourceModeNegative</strong></a></li></ul><h3><a name='org.jboss.windup.tests:tests' /><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.tests$tests/testReport'>org.jboss.windup.tests:tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='https://10.3.9.192/job/windup-pr-builder/408/org.jboss.windup.tests$tests/testReport/org.jboss.windup.tests.application/GraphServiceLookupTest/testServiceLookup/'><strong>org.jboss.windup.tests.application.GraphServiceLookupTest.testServiceLookup</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Merged build started.

**Comment by windupgithubbot1** [comment]
Merged build finished. Test PASSed.

**Comment by windupgithubbot1** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://10.3.9.192//job/windup-pr-builder/409/
Test PASSed.

**Pull request closed without merging**